framework/src/vectorpostprocessors/WorkBalance.C

```diff
@@ -193,6 +193,8 @@ public:
 
       // Build the side so we can compute its volume
       auto side_elem = elem->build_side_ptr(side);
+      // FIXME: I don't know how to get the volume here, there is hardly anything to
+      // work with in this threaded loop
```

Comment:
I need help to fix this instance.

Reply:
You should have `_assembly` available via `UserObject`. Then, you should be able to grab `sideElemVolume()`. The side elem volume should be computed, becuase `reinit` should have been called before we call `onInternalSide`.

This is a general concept that should work in almost every MOOSE system. Things like `Function`s would be example where that would not be the case. So you should be able to use this fix in the other instances as well...

Reply:
This piece of code is not part of the VPP. It's part of 

```
class WBElementLoop : public ThreadedElementLoopBase<ConstElemRange>
```

which does not have assembly available.

Reply:
You are right. Now I see it.

Then, pass `_fe_problem` into `WBElementLoop` like it is done for `WBNodeLoop` and get the `Assembly` out of that in the c-tor.

Reply:
Just talked to @friedmud about this. Leave it as is (side volume from libMesh). We want to know the physical (as in mesh physical) side areas here for the purposes of balancing.

Reply:
`elem->volume()` is not the physical side area in RZ, whatever Assembly has is. 
But if that is what you want, I'll leave it.